docs/user-guide/troubleshooting.md

```diff
@@ -11,60 +11,75 @@ Do not move it without providing redirects.
 
 ## Authentication to the Kubernetes API Server
 
-If your Kubernetes cluster is not configured correctly, it may fail to contact
-the API server. Before
-explaining the solution, it is useful to review how the dashboard discovers
-and authenticates with the API server.
+
 A number of components are involved in the authentication process and the first step is to narrow
 down the source of the problem, namely whether it is a problem with user authentication or with service authentication.
-In the diagram below you can see the full authentication flow, starting with the browser
+Both authentications must work:
+
+```
++-------------+   user             +-------------+   service          +-------------+
+|             |   authentication   |             |   authentication   |             |
+|  browser    +------------------->+  apiserver  +<-------------------+  dashboard  |
+|             |                    |             |                    |             |
++-------------+                    +-------------+                    +-------------+
+
+```
+
+__User authentication__
+
+From your workstation you can connect to Dashboard in three different ways:
+
+1. _Authentication through kubectl:_ This is recommended, because it is secure and easy. In your desktop environment (e.g. laptop)
+verify that kubectl is configured properly with `kubectl cluster-info`. If you get a response then you can continue.
+Enter the command `kubectl proxy`. kubectl will authenticate with the apiserver and proxy every request to a local address which is unprotected. This is not a problem as the site is only accessible locally. Now access Dashboard with `http://localhost:8001/ui`. If it fails then your problem is located with service authentication. See next section.
+
+2. _Direct access to apiserver:_ Open a browser with the URL `https://<master>/`. A login dialog should pop up. If it does not, then username & password authentication is not configured for the apiserver. See [documentation](http://kubernetes.io/docs/admin/authentication/) if you want to configure it manually. Next, access Dashboard with `https://<master>/ui`. If it fails then your problem is located with service authentication. See next section.
+
+3. _Bypass apiserver:_ If you are working in a trusted environment then you may access Dashboard without authentication. Expose Dashboard service via NodePort. See [user guide](http://kubernetes.io/docs/user-guide/services/) for details.
+
+
+
+__Service authentication__
+
+Dashboard needs information from apiserver. Therefor, it must authenticate, which can be achieved in two different ways:
```

Comment:
Therefor --> Therefore 
authenticate something or someone = check the authenticity of something or someone, 
here do you mean 'pass authentication checks (imposed by the apiserver)' or  'must authenticate [something or someone]'? In the latter case say what or whom as the verb is transitive.